cfpb/cfgov-refresh
# Add V1Page model and hooks for 2 site publishing

# Here we go
This is the work to prove the second hypothesis, which states that we should be able to publish to a `content` site separately from production and that the content on the `content` site should be viewable from within the bureau only without the need for a user account. Phew.

## A couple of things to mention before you start
You should delete your sqlite database file first. It's just easier that way. Then you have to run `python cfgov/manage.py migrate` before you start to set up the database. After that, run `createsuperuser`.

**THIS STEP IS NECESSARY**
Go to Settings, click Sites, and then add a site with the hostname `content.localhost` with the same settings as the `localhost` site, except change both of the sites to point to port 8000.

Test by:
- creating a Blog Page under the site's homepage and saving it as a draft. 
- Go to `content.localhost:8000/blog-post-slug` and see if it's there. No? Good. Do the same thing for `localhost:8000/blog-post-slug`.
- Go back to editing the page and then click `Share on Content`. Check the content site. There? Good. Check `localhost`. There? Bad.
- Repeat last step for `Publish to WWW`. Both sites should render the page.
- Do reverse steps to test out the `Unshare` function (you should see buttons on the list view as well as the page you're editing on)

The actual template for the page is simple. Only temporary. _The URL structure isn't implemented yet._ This only adds the ability to `Share on Content`. 

- @kave 
- @rosskarchner
